csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -631,13 +639,14 @@ std::unique_ptr<ReductionParams> innerOuterPersistentHeuristic(
   rparams->vectorize_inner_reduction = iop.inner_vect > 1;
   rparams->split_grid_dim_iter_dom_outer = true;
   rparams->grid_dim_iter_dom = ParallelType::BIDy;
+
   rparams->lparams = LaunchParams(
       LaunchParams::UNINITIALIZED_VAL,
       iop.gdimy,
       LaunchParams::UNINITIALIZED_VAL,
       iop.bdimx,
       iop.bdimy,
-      LaunchParams::UNINITIALIZED_VAL);
+      iop.bdimz);
```

Comment:
Naive question regarding iop.bdimz. Why do we add this in heuristic if it's always going to be 1?

Reply:
It is redundant since we already have `NVF_ERROR(iop.bdimz == 1, "bdimz must be 1.");`
Removed this change.